rotkehlchen/api/v1/schemas.py

```diff
@@ -1921,7 +1921,7 @@ class IdentifiersListSchema(Schema):
 
 
 class AssetsImportingSchema(Schema):
-    file = FileField(allowed_extensions=['.json'], load_default=None)
+    file = FileField(allowed_extensions=['.zip'], load_default=None)
```

Comment:
Why not process both? The .zip thing is since kelsos asked for it so that he and all other users can just export and import, right?

Makes sense.

But would it not also make sense to simply import a single json file?

Reply:
I've allow for both. Wanted to avoid that too nested logic that you hate and didn't see a better way than what I pushed